dev/com.ibm.ws.org.apache.myfaces.2.3/build.gradle

```diff
@@ -1,41 +0,0 @@
-/*******************************************************************************
```

Comment:
We should probably do any JSF 2.3 work as part of the 2.3  third party PR

Reply:
This file shouldn't have made it into the commit